modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties

```diff
@@ -20659,6 +20659,7 @@ visit-our-privacy-policy=Visit our Privacy Policy
 visited-link=Visited Link
 visitors=Visitors
 visits=Visits
+visualization-modes=Visualization Modes
```

Comment:
It's fine to run `gradle buildLang`, but you don't need to. It may save you some conflict annoyances. It's enough to just add the change to `Language.properties`.

Reply:
:scream: :scream: :scream: I've been adding the buildLang commit all these years, and it is not needed at all?

Reply:
Nope :slightly_smiling_face:  

I only do it if I want to double-check locally, that I didn't make a typo in lang key or something like that. After that, I discard auto-translations.